### PR TITLE
update links, text, no analytics

### PR DESCRIPTION
**Pull request by kkraune**

### CODE REVIEW

**Comment by kkraune** [comment]
I'll make more changes, converting to draft

File: container-search-gui/src/main/resources/gui/_includes/index.html [hunk 1]
```diff
@@ -137,7 +137,7 @@
                     </br>
                     <div class="intro-param" id="help">
                       <div class="help-title">Help</div>
-                      <span> If you find errors, spelling mistakes, faulty pieces of code or want to improve the querybuilder, please submit a pull request or create an <a href="https://github.com/vespa-engine/vespa/issues">issue</a>.</span> </br>
+                      <span>Please submit a pull request or create an <a href="https://github.com/vespa-engine/vespa/issues">issue</a> for fixes to the querybuilder.</span> </br>
```
Comment (ean) [inline container-search-gui/src/main/resources/gui/_includes/index.html:140]:
```suggestion
                      <span>Please submit a <a href="https://github.com/vespa-engine/vespa/blob/master/container-search-gui/src/main/resources/gui/_includes/index.html">pull request</a> or create an <a href="https://github.com/vespa-engine/vespa/issues">issue</a> for fixes to the querybuilder.</span> </br>
```

**Review by ean — APPROVED** [review]
added a small suggestion for change to reduce the friction of changing errors in the document.

**Comment by kkraune** [comment]
Did some cleanup, removed the feature to look up in the query api reference - this was broken, using a local file. That code is just commented out if we want to look up in the real site, which is very doable. For next PR

had to reformat at fix a lot of IDE warnings, too, so PR not that readable - this is just a stop-gap. Works on my computer :-)

querybuilder

**Comment by kkraune** [comment]
We will leave this for now and merge post-8, no rush. @ldalves FYI as well, we can live with this short term until we think of something better. we can use this as a basis

**Comment by kkraune** [comment]
for reference, current site:

qb

**Comment by kkraune** [comment]
for the record, the new look posted above is added in https://github.com/vespa-engine/vespa/pull/23111

**Pull request merged**